builder/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/rohansingh9001/Neura-Launch-Dashboard/builder/controllers"
+)
+
+func main() {
+	r := gin.Default()
+
+	// Routes
+	r.GET("/", controllers.PingController)
+	r.GET("/verifyCheckSum", controllers.VerifyCheckSumController)
+	r.GET("/downloadFiles", nil)
+	r.GET("/buildImage", nil)
+
+	// Starting the server
+	r.Run(":8000")
+}
```

Comment:
U have exposed port 5000 in the docker build dev file but you have started the service on port 8000 ?

Reply:
The container is listening on PORT 5000, while the application is listening on PORT 8000. This should be correct afaik.